extensions/sql-database-projects/src/common/constants.ts

```diff
@@ -61,6 +61,7 @@ export const newDefaultProjectSaveLocation = localize('newDefaultProjectSaveLoca
 export const invalidDefaultProjectSaveLocation = localize('invalidDefaultProjectSaveLocation', "Default location to save new database projects is invalid. Would you like to update it?");
 export const openWorkspaceSettings = localize('openWorkspaceSettings', "Yes, open Settings");
 export const doNotPromptAgain = localize('doNotPromptAgain', "Don't ask again");
+export const refreshProject = localize('refreshProject', "Would you like to refresh your database project?");
```

Comment:
I think "reload" is typically the verb used here since "refresh" can be ambiguous in which direction it goes.